CorePayments/src/main/java/com/paypal/android/corepayments/analytics/AnalyticsEventData.kt

```diff
@@ -2,7 +2,6 @@ package com.paypal.android.corepayments.analytics
 
 // Ref: https://blog.klipse.tech/databook/2022/06/22/separate-code-from-data.html
 data class AnalyticsEventData(
-    val clientID: String,
```

Comment:
We still want to send `clientID` in our analytics. We should be sending the value that the merchant provides. The clientID is our way of tying events to specific merchants.

Reply:
Yup! We still send it [here](https://github.com/paypal/Android-SDK/blob/449aeabf1c1d9d2354616402bdeeeba5f75d9a7a/CorePayments/src/main/java/com/paypal/android/corepayments/TrackingEventsAPI.kt#L36).